packages/hop-node/src/watchers/classes/BaseWatcher.ts

```diff
@@ -95,23 +97,20 @@ class BaseWatcher extends EventEmitter implements IBaseWatcher {
   }
 
   async preSyncHandler () {
-    this.logger.debug('syncing up events')
+    this.syncIndex++
```

Comment:
i think the incrementer should be moved to the post handler before the wait (so it starts at 0)

Reply:
Fix 1b4fc481ea7398f99fbd7364d4ae3a6a38eaf4dc